src/plugins/info.cr

```diff
@@ -4,7 +4,7 @@ class TournamentBot::Info
 
   @[Discord::Handler(event: :message_create, middleware: Command.new("!info"))]
   def info(payload, _ctx)
-    bot = client.cache.try &.resolve_user(CLIENT_ID)
+    bot = client.cache.try &.resolve_user(TournamentBot.config.client_id)
```

Comment:
Worth noting: `Cache#resolve_current_user` (this kind of request does not need client_id)

 https://meew0.github.io/discordcr/doc/master/Discord/Cache.html#resolve_current_user%3AUser-instance-method

Reply:
I mean, that sounds like the smart thing to do. I must've overlooked this while scrolling through the docs.